models/place-add-video.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "place.add.video",
```

Comment:
I'd prefer if this followed the convention `place.{property}.{method}` like for example the existing `place.calendar.put`. It's more descriptive than "add", which helps when we use this file in the actual code to validate a request (that uses a specific HTTP method). So in this case it would be `place.videos.post`.

The filename should then also be `place-{property}-{method}`, for example `place-videos-post.json`.

Reply:
Indeed a very clean naming convention. I changed it for both place and event.